specification/workloads/resource-manager/Microsoft.Workloads/SAPVirtualInstance/preview/2023-10-01-preview/SAPVirtualInstance.json

```diff
@@ -0,0 +1,4455 @@
+          },
+          "Start Virtual Machine and the SAP Application Server Instance on it.": {
+            "$ref": "./examples/sapapplicationinstances/SAPApplicationServerInstances_StartInstance_WithInfraOperations.json"
+          }
+        },
+        "description": "Starts the SAP Application Server Instance.",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SAPVirtualInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SAPApplicationInstanceNameParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "SAP Application server instance start request body.",
+            "schema": {
+              "$ref": "#/definitions/StartRequest"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "The URI to fetch the updated Virtual Instance for SAP resource."
+              }
+            }
+          },
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v3/types.json#/definitions/OperationStatusResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Workloads/sapVirtualInstances/{sapVirtualInstanceName}/applicationInstances/{applicationInstanceName}/stop": {
+      "post": {
+        "tags": [
+          "SAPApplicationServerInstances"
+        ],
+        "operationId": "SAPApplicationServerInstances_StopInstance",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Stop the SAP Application Server Instance": {
+            "$ref": "./examples/sapapplicationinstances/SAPApplicationServerInstances_StopInstance.json"
+          },
+          "Stop the SAP Application Server Instance and it's infrastructure.": {
```

Comment:
Rephrase this as 
Stop the SAP Application Server Instance and the Virtual machine.